std/conv.d

```diff
@@ -3926,6 +3926,8 @@ if (is(UT == Unqual!T))
     {
         static assert (is(typeof({static T i;})),
             convFormat("Cannot emplace a %1$s because %1$s.this() is annotated with @disable.", T.stringof));
+        static if (is(T == class)) static assert (!isAbstractClass!T,
+            T.stringof ~ " is abstract and it can't be emplaced");
```

Comment:
The static-if test is redundant (and `static if (…) { static assert(…); }` can always be coalesced into just `static assert(…);`)